app/controllers/v0/messages_controller.rb

```diff
@@ -78,7 +84,7 @@ def move
     private
 
     def message_params
-      params.require(:message).permit(:id, :category, :body, :recipient_id, :subject)
+      @message_params ||= params.require(:message).permit(:id, :category, :body, :recipient_id, :subject)
```

Comment:
don't need to permit id as part of message, this is explicitly required as part of the route / uri string